### PR TITLE
ACMS-1608: Updating configurations for ACMS modules

### PR DESCRIPTION
**Pull request by deepakmishra2**
**Motivation**

Fixes ACMS-1608

### CODE REVIEW

File: modules/acquia_cms_page/config/optional/node.type.page.yml [hunk 1]
```diff
@@ -1,17 +1,23 @@
+uuid: b8e80b75-8f73-4536-9b41-b25265bfae22
```
Comment (rajeshreeputra) [inline modules/acquia_cms_page/config/optional/node.type.page.yml:1]: this need to be removed

File: modules/acquia_cms_page/config/optional/node.type.page.yml [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
 langcode: en
 status: true
 dependencies:
+  module:
+    - acquia_cms_common
+    - menu_ui
+    - node_revision_delete
```
Comment (rajeshreeputra) [inline modules/acquia_cms_page/config/optional/node.type.page.yml:8]: node revision delete is not a dependency in page, remove this

File: modules/acquia_cms_page/config/optional/node.type.page.yml [hunk 3]
```diff
@@ -37,6 +43,12 @@ third_party_settings:
     unpublish_enable: true
     unpublish_required: false
     unpublish_revision: false
+  node_revision_delete:
```
Comment (rajeshreeputra) [inline modules/acquia_cms_page/config/optional/node.type.page.yml:46]: node revision delete is not a dependency in page, remove this

File: modules/acquia_cms_page/config/optional/node.type.page.yml [hunk 4]
```diff
@@ -37,6 +43,12 @@ third_party_settings:
...
+    minimum_revisions_to_keep: 30
+    minimum_age_to_delete: 0
+    when_to_delete: 0
+_core:
```
Comment (rajeshreeputra) [inline modules/acquia_cms_page/config/optional/node.type.page.yml:50]: remove _core hash

File: modules/acquia_cms_person/config/optional/core.entity_form_display.node.person.default.yml [hunk 5]
```diff
@@ -1,3 +1,4 @@
+uuid: eb2d2fbd-fa47-4fe6-9d6b-f608a4dd72b7
```
Comment (rajeshreeputra) [inline modules/acquia_cms_person/config/optional/core.entity_form_display.node.person.default.yml:1]: remove uuid

File: modules/acquia_cms_place/config/optional/views.view.places_fallback.yml [hunk 6]
```diff
@@ -1,12 +1,16 @@
...
 status: true
 dependencies:
   config:
     - core.entity_view_mode.node.teaser
     - node.type.place
   module:
+    - cohesion
     - node
     - user
+_core:
```
Comment (rajeshreeputra) [inline modules/acquia_cms_place/config/optional/views.view.places_fallback.yml:12]: remove _core hash

File: modules/acquia_cms_place/config/optional/views.view.places_fallback.yml [hunk 7]
```diff
@@ -1,12 +1,16 @@
+uuid: fb5cd577-47ca-431e-968e-41e4ea7af862
```
Comment (rajeshreeputra) [inline modules/acquia_cms_place/config/optional/views.view.places_fallback.yml:1]: remove uuid

File: modules/acquia_cms_place/config/optional/views.view.places_fallback.yml [hunk 8]
```diff
@@ -1,12 +1,16 @@
...
 langcode: en
 status: true
 dependencies:
   config:
     - core.entity_view_mode.node.teaser
     - node.type.place
   module:
+    - cohesion
```
Comment (rajeshreeputra) [inline modules/acquia_cms_place/config/optional/views.view.places_fallback.yml:9]: remove cohesion module dependency

File: modules/acquia_cms_article/config/optional/views.view.articles.yml [hunk 9]
```diff
@@ -6,7 +6,9 @@ dependencies:
     - search_api.index.content
     - views.view.articles_fallback
   module:
+    - acquia_cms_common
     - acquia_cms_search
+    - cohesion
```
[the remark this answers is not in the document]
Comment (deepakmishra2) [reply modules/acquia_cms_article/config/optional/views.view.articles.yml]: changes are done

File: modules/acquia_cms_page/config/optional/node.type.page.yml (hunk 1 above)
Comment (deepakmishra2) [reply modules/acquia_cms_page/config/optional/node.type.page.yml]: changes are done

File: modules/acquia_cms_person/config/optional/core.entity_form_display.node.person.default.yml (hunk 5 above)
Comment (deepakmishra2) [reply modules/acquia_cms_person/config/optional/core.entity_form_display.node.person.default.yml]: changes are done

File: modules/acquia_cms_place/config/optional/views.view.places_fallback.yml (hunk 6 above)
Comment (deepakmishra2) [reply modules/acquia_cms_place/config/optional/views.view.places_fallback.yml]: changes are done

File: modules/acquia_cms_place/config/optional/views.view.places_fallback.yml (hunk 7 above)
Comment (deepakmishra2) [reply modules/acquia_cms_place/config/optional/views.view.places_fallback.yml]: changes are done

File: modules/acquia_cms_place/config/optional/views.view.places_fallback.yml (hunk 8 above)
Comment (deepakmishra2) [reply modules/acquia_cms_place/config/optional/views.view.places_fallback.yml]: changes are done

File: modules/acquia_cms_page/config/optional/node.type.page.yml (hunk 4 above)
Comment (deepakmishra2) [reply modules/acquia_cms_page/config/optional/node.type.page.yml]: changes are done

File: modules/acquia_cms_page/config/optional/node.type.page.yml (hunk 2 above)
Comment (deepakmishra2) [reply modules/acquia_cms_page/config/optional/node.type.page.yml]: changes are done

**Pull request merged**